docs/release-notes.md

```diff
@@ -32,6 +32,19 @@ updates as well.
   2.9.10 has an infinite loop in a certain end-of-file situation." Fix
   by updating a dependency in documentation generation.
 
+### Zephyr-RTOS compatibility
+- for compatibility with zephyr 2.2.0 need to revert commits which align MCUBoot
+  to the zephyr master branch at the release time. These commits are reverted on
+  1.6.0-revert-zephyr-2.2.0-compatible branch.
```

Comment:
this branch (name might be change) must be pushed to to this repo within the release process.

Reply:
Yeah, that branch name is just a little bit wordy. Revert and compatible are kind of redundant. I'd just call it v1.6.0-zephyr-2.2.x (or 2.2.1, which ever one is current when that happens.